README.md

```diff
@@ -24,50 +24,66 @@ It offers:
 
 <ul>
 <li>
-    🔧 Plug ‘n’ play composable pipelines for creating datasets for
+    🔧 Plug ‘n’ play composable pipelines for creating datasets for:
     <ul>
         <li>AI image generation model fine-tuning (Stable Diffusion, ControlNet)</li>
         <li>Large language model fine-tuning (LLaMA, Falcon)</li>
     </ul>
 </li>
 <li>
-🧱 Library of off-the-shelf reusable components for
+🧱 Off-the-shelf library of easily adaptable reusable components for:
     <ul>
-    <li>Extracting data from public sources such as Common Crawl, LAION, ...</li>
-    <li>Filtering on 
+    <li>
+    Data Filtering
     <ul>
-        <li>Content, e.g. language, visual style, topic, format, aesthetics, etc.</li>
-        <li>Context, e.g. copyright license, origin</li>
-        <li>Metadata</li>
+        <li>Content: language, visual style, topic, format, aesthetics, NSFW, etc.</li>
+        <li>Context: e.g. copyright license, origin, etc.</li>
+        <li>Deduplication</li>
+    </ul>
+    </li>
+    <li>
+    Data Enrichment
+    <ul>
+        <li>Captions</li>
+        <li>Segmentations</li>
+        <li>Embeddings</li>
+    </ul>
+    </li>
+    <li>
+    Data Transformation
+    <ul>
+        <li>Images: cropping, resizing, etc. </li>
+        <li>Text: chunking, indexing, etc. </li>
+    </ul>
+    </li>
+    <li>
+    Data retrieval
+    <ul>
+        <li>Common Crawl</li>
+        <li>LAION</li>
     </ul>
     </li>
-    <li>Removal of unwanted data such as toxic, NSFW or generated content</li>
-    <li>Removal of unwanted data patterns such as societal bias</li>
-    <li>Transforming data (resizing, cropping, reformatting, …)</li>
-    <li>Tuning the data for model performance (normalization, deduplication, …)</li>
-    <li>Enriching data (captioning, metadata generation, synthetics, …)</li>
-    <li>Transparency, auditability, compliance</li>
     </ul>
 </li>
 <li>📖 🖼️ 🎞️ ♾️ Out of the box multimodal capabilities: text, images, video, etc.</li>
-<li>🐍 Standardized, Python/Pandas-based way of creating custom components</li>
-<li>🏭 Production-ready, scalable deployment</li>
-<li>☁️ Multi-cloud integrations</li>
+<li>🐍 A simple Pandas based interface for creating custom components</li>
+<li>🏭 Scalable deployment for production environments.</li>
```

Comment:
I would keep `production-ready` here, as it's more of a USP of Fondant than `scalable`.